deploy.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+# This script is not currently meant to be executed from within the repository.
+# Copy or symlink it to some deployment directory.
+
+set -ex
+
+# Defaults for production.
+# Example for development:
+#   PORT=8100 OPENFISCA_PORT=2000 OPENFISCA_CONFIG=development PUBLIC_URL=http://next.mes-aides.sgmap.fr ./deploy.sh aah
+TARGET_BRANCH=${1:-master}  # demo
+PORT=${PORT:-8000}  # 8100
+OPENFISCA_PORT=${OPENFISCA_PORT:-12000}  # 2000
+OPENFISCA_CONFIG=${OPENFISCA_CONFIG:-mes-aides}  # development
+PUBLIC_URL=${PUBLIC_URL:-https://mes-aides.gouv.fr}  # http://next.mes-aides.sgmap.fr
+
+# Install Mes Aides
+if ! cd mes-aides-ui
+then
+    git clone https://github.com/sgmap/mes-aides-ui.git
+    cd mes-aides-ui
+fi
+
+git fetch origin $TARGET_BRANCH
+git checkout origin/$TARGET_BRANCH
+
+rm -rf node_modules # work around libsass bindings being never found
+npm install
+grunt build
+
+# Stop Mes Aides
+killall --user `whoami` node || echo 'No server was running'
+# Start Mes Aides
+OPENFISCA_URL="http://localhost:$OPENFISCA_PORT" SESSION_SECRET=foobar NODE_ENV=production MES_AIDES_ROOT_URL="$PUBLIC_URL" PORT=$PORT MONGODB_URL="mongodb://localhost/$(whoami)" nohup node server.js >> ../mes-aides_log.txt &
+
+cd ..
+
+# Install OpenFisca
+if ! cd openfisca
+then
+    git clone https://github.com/sgmap/openfisca.git
+    cd openfisca
+fi
+
+git fetch origin $TARGET_BRANCH
+git checkout origin/$TARGET_BRANCH
+
+./update.sh --dev
```

Comment:
Right, and remark is 100% apt too. I felt it when writing.

Do we change the default `update.sh` behaviour and make it stay on the current branch? I made it default to switch to `master` to ensure braindead calls would always _actually_ update to the latest stable version, not do nothing just because you were on a stale branch, but I end up almost always call it with `--dev`.

Options:
1. Always use current branch.
2. Rename `--dev` option to something like `--stay-on-branch` or `--current-branch`.
3. Require a target branch as a parameter (and allowing something like `--current-branch` as a valid target to avoid having to figure it out from git all the time).